eventsources/sources/amqp/start.go

```diff
@@ -85,13 +85,13 @@ func (el *EventListener) StartListening(ctx context.Context, dispatch func([]byt
 		if amqpEventSource.Auth != nil {
 			username, err := common.GetSecretFromVolume(amqpEventSource.Auth.Username)
 			if err != nil {
-				return errors.Wrap(err, "username not founnd")
+				return errors.Wrap(err, "username not found")
 			}
 			password, err := common.GetSecretFromVolume(amqpEventSource.Auth.Password)
 			if err != nil {
-				return errors.Wrap(err, "password not founnd")
+				return errors.Wrap(err, "password not found")
 			}
-			c.SASL = []amqplib.Authentication{&amqplib.AMQPlainAuth{
+			c.SASL = []amqplib.Authentication{&amqplib.PlainAuth{
```

Comment:
Changing to the latter seems to work.

I was able to find descriptions of AMQPLAIN and PLAIN here: https://www.rabbitmq.com/access-control.html
Apparently, AMQPLAIN (which we were using) is the non-standard version of PLAIN.